### PR TITLE
[pt-br] Adding translation of ports and protocols page

### PR DESCRIPTION
**Pull request by MarcosNBJ**
# Changes
* Brazilian portuguese translation of the [Ports And Protocols](https://kubernetes.io/docs/reference/ports-and-protocols/) page.

## Required steps
- [x] Content translated 
- [x] Links to other pages with translations updated 
## Related Issue
#13939

### CODE REVIEW

**Comment by devlware** [comment]
Sorry for the late reply @MarcosNBJ, I'll review later today!

File: content/pt-br/docs/reference/ports-and-protocols.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+weight: 50
+---
+
+Quando o Kubernetes está sendo executado em um ambiente com uma rede mais restritiva, 
+como por exemplo um data center on-premises com firewalls de rede físicos ou redes virtuais em nuvens públicas,
+é útil saber quais portas e protocolos são utilizados pelos componentes do Kubernetes. 
+
+## Camada de gerenciamento
+
+| Protocol | Direction | Port Range | Purpose                 | Used By                   |
```
Comment (devlware) [inline content/pt-br/docs/reference/ports-and-protocols.md:13]: @MarcosNBJ Maybe we should translate the description names of each column.

File: content/pt-br/docs/reference/ports-and-protocols.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+|----------|-----------|------------|-------------------------|---------------------------|
+| TCP      | Entrada   | 6443       | Servidor da API do Kubernetes | Todos                       |
+| TCP      | Entrada   | 2379-2380  | API servidor-cliente do etcd  | kube-apiserver, etcd      |
+| TCP      | Entrada   | 10250      | API do kubelet             | kubeadm, Camada de gerenciamento       |
+| TCP      | Entrada   | 10259      | kube-scheduler          | kubeadm                      |
+| TCP      | Entrada   | 10257      | kube-controller-manager | kubeadm                      |
+
+Embora as portas do etcd estjam inclusas na secção da Camada de gerenciamento, você também
```
Comment (devlware) [inline content/pt-br/docs/reference/ports-and-protocols.md:21]:
```suggestion
Embora as portas do etcd estejam inclusas na secção da Camada de gerenciamento, você também 

```

File: content/pt-br/docs/reference/ports-and-protocols.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+| TCP      | Entrada   | 10250      | API do kubelet             | kubeadm, Camada de gerenciamento       |
+| TCP      | Entrada   | 10259      | kube-scheduler          | kubeadm                      |
+| TCP      | Entrada   | 10257      | kube-controller-manager | kubeadm                      |
+
+Embora as portas do etcd estjam inclusas na secção da Camada de gerenciamento, você também 
+pode hospedar o seu próprio cluster etcd externamente ou em portas customizadas. 
+
+## Nós de processamento {#node}
+
+| Protocol | Direction | Port Range  | Purpose               | Used By                 |
```
Comment (devlware) [inline content/pt-br/docs/reference/ports-and-protocols.md:26]: Same comment here on the column names.

**Review by devlware — CHANGES_REQUESTED** [review]
Left some comments Marcos.

File: content/pt-br/docs/reference/ports-and-protocols.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+| Protocolo | Direção | Intervalo de Portas | Propósito                 | Utilizado por                   |
+|----------|-----------|------------|-------------------------|---------------------------|
+| TCP      | Entrada   | 6443       | Servidor da API do Kubernetes | Todos                       |
+| TCP      | Entrada   | 2379-2380  | API servidor-cliente do etcd  | kube-apiserver, etcd      |
+| TCP      | Entrada   | 10250      | API do kubelet             | kubeadm, Camada de gerenciamento       |
+| TCP      | Entrada   | 10259      | kube-scheduler          | kubeadm                      |
+| TCP      | Entrada   | 10257      | kube-controller-manager | kubeadm                      |
+
+Embora as portas do etcd estejam inclusas na secção da Camada de gerenciamento, você também
```
[the remark this answers is not in the document]
Comment (MarcosNBJ) [reply content/pt-br/docs/reference/ports-and-protocols.md]: Woops. Nice catch. Fixed

**Comment by edsoncelio** [comment]
@devlware this PR is done for merge (i think), could you take a look please?

**Pull request merged**